src/sections/HomeServices/index.js

```diff
@@ -3,35 +3,69 @@ import PropTypes from "prop-types";
 import Section from "root/components/Section";
 import Typography from "root/components/Typography";
 import ParallaxEffect from "root/components/ParallaxEffect";
+import updateDimensions from "root/hooks/useWindowDimensions";
 import { map } from "lodash-es";
 import classNames from "classnames";
+import { Parallax } from "react-scroll-parallax";
 
 import withQuery from "./withQuery";
 
 import "./index.css";
 
+const breakpointMobile = 800;
+
 const HomeServices = ({ data }) => {
-  const renderBlock = (title, subtitle, points) => {
+  const dimensions = updateDimensions();
+
+  const renderBlock = (title, subtitle, points, parallaxY, parallaxX) => {
```

Comment:
Can you be more explicit with the `points` parameter?

Reply:
✅